experimental/PropertyDDS/packages/property-common/src/error_objects/httpErrorNoStack.ts

```diff
@@ -3,8 +3,12 @@
  * Licensed under the MIT License.
  */
 
-import { FlaggedError } from "./flaggedError";
-import { HTTPError } from "./httpError";
+import { FlaggedError } from "./flaggedError.js";
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { HTTPError } from "./httpError.js";
```

Comment:
```suggestion
import { HTTPError } from "./httpError.js";
```

Reply:
before adding this line into the code, it was failing `npm run checks`